### PR TITLE
profile privacy BL

### PR DESCRIPTION
**Pull request by Gusvarov**
**task [1008](https://github.com/ita-social-projects/GreenCity/issues/1008)**

added posibility to choose profile privacy
image
for example, choose: 'showEcoPlaces' and on profile we will display eco places
image
If we choose one more option, it will be displayed in profile, for example shopping list
image

### CODE REVIEW

**Review by yulkats — APPROVED** [review]

File: src/app/component/user/components/profile/edit-profile/edit-profile.component.scss [hunk 1]
```diff
@@ -20,6 +20,73 @@ $main-font: OpenSans, sans-serif;
...
+                    input:checked ~ .checkmark:after {
+                        display: block;
+                    }
+        
+                    .checkmark {
+                        border: 1px solid #13AA57;
+                        border-radius: 4px;
+                        height: 20px;
+                        left: 0;
+                        position: absolute;
```
Comment (mr-pavlenko) [inline src/app/component/user/components/profile/edit-profile/edit-profile.component.scss:66]: Why did you use the position `absolute`?

Comment (Gusvarov) [reply src/app/component/user/components/profile/edit-profile/edit-profile.component.scss:66]: It's actually not my code, but as i can see ti have to stylize a checkbox, so it hide input and replaced it with span

**Review by HryshkoAnton — APPROVED** [review]

**Review by sofiiamatsyhin — APPROVED** [review]

**Comment by mplesha** [comment]
your code has some conflicts

File: src/app/component/user/user.module.ts [hunk 2]
```diff
@@ -49,7 +49,6 @@ import {
   EditProfileComponent,
   PersonalPhotoComponent,
   SocialNetworksComponent,
-  ProfilePrivacyComponent
 } from './components';
 import { CommonModule } from '@angular/common';
```
Comment (mplesha) [inline src/app/component/user/user.module.ts:53]: -- `angular` import between components.
So, organize imports:
order `angular` imports first, next other third party libs and only then imports of your application.

Comment (Gusvarov) [reply src/app/component/user/user.module.ts]: fixed

**Comment by Gusvarov** [comment]
> your code has some conflicts

fixed

**Comment by mplesha** [comment]
conflicts

**Pull request merged**